lib/propagation/util.js

```diff
@@ -7,3 +7,19 @@ exports.currentSpanId = currentSpanId;
 function currentSpanId(context) {
   return context.stack[context.stack.length - 1].payload[schema.TRACE_SPAN_ID];
 }
+
+exports.getPropagationContext = getPropagationContext;
+
+// getPropagationContext accepts the current execution context async hook
```

Comment:
```suggestion
// getPropagationContext accepts the current execution context
```

async_hooks is the facility we use to transmit the context around.